.github/workflows/test-code.yml

```diff
@@ -45,7 +42,7 @@ jobs:
       #   run: cargo +${{ env.rustc_version }} clippy -- -D warnings
 
       - name: Check Code
-        run: cargo check
+        run: cargo check --release
 
       - name: Test Code
         run: cargo test
```

Comment:
Should it be also 
Check and Test Code
?

Reply:
No, two separate actions here.